Error loading the plugin after updating to v0.26.0

@afharo thank you for implementing the dust bin that fast! You're on fire!
Sadly it now throws an error at me:

```
[9/1/2022, 7:01:35 AM] ====================
[9/1/2022, 7:01:35 AM] ERROR LOADING PLUGIN homebridge-xiaomi-roborock-vacuum:
[9/1/2022, 7:01:35 AM] Error: Cannot find module 'hap-nodejs'
Require stack:
- /homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/dist/services/custom_care_service/care_service.js
- /homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/dist/services/custom_care_service/index.js
- /homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/dist/services/care_service.js
- /homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/dist/services/index.js
- /homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/dist/xiaomi_roborock_vacuum_accessory.js
- /homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/dist/index.js
- /homebridge/node_modules/homebridge/lib/plugin.js
- /homebridge/node_modules/homebridge/lib/pluginManager.js
- /homebridge/node_modules/homebridge/lib/server.js
- /homebridge/node_modules/homebridge/lib/cli.js
- /homebridge/node_modules/homebridge/bin/homebridge
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/homebridge/node_modules/homebridge-xiaomi-roborock-vacuum/src/services/custom_care_service/care_service.ts:1:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
[9/1/2022, 7:01:35 AM] ====================
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,8 @@
 
 ## Next
 
+- [Bug] Add dependency to `hap-nodejs` (#531).
+
 ## 0.26.0
 
 - [Enhancement] Add Dust Bin needs to be emptied indicator (#111).

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "chalk": "4.1.2",
     "debug": "4.3.4",
     "deep-equal": "2.0.5",
+    "hap-nodejs": "^0.10.2",
     "rxjs": "^7.5.6",
     "semver": "7.3.7",
     "tinkerhub-discovery": "0.3.1",

